worldedit-fabric/src/main/java/com/sk89q/worldedit/fabric/FabricWorldEdit.java

```diff
@@ -284,6 +284,12 @@ private ActionResult onRightClickBlock(PlayerEntity playerEntity, World world, H
 
     // TODO Pass empty left click to server
 
+    // TODO: Use the fabric-networking-api-v1 event when updating
```

Comment:
Would you mind just updating to v1 for us? Or I can do it separately.

Reply:
Sure, I've updated to networking-v1